lib/modules/showImages.js

```diff
@@ -376,38 +397,22 @@ const resizeSources = {
  */
 function enableConserveMemory() {
 	window.addEventListener('scroll', _.debounce(() => {
-		const bufferScreens = module.options.bufferScreens.value || 2;
-		const viewportHeight = window.innerHeight;
-		const maximumTop = viewportHeight * (bufferScreens + 1);
-		const minimumBottom = viewportHeight * bufferScreens * -1;
-
-		const isWithinBuffer = ele => {
-			const { bottom, top } = ele.getBoundingClientRect();
-			return top <= maximumTop && bottom >= minimumBottom;
-		};
+		const activeExpandos = Array.from(primaryExpandos).map(([, expando]) => expando);
```

Comment:
This (and other places where `Object.values()` was replaced) could be changed to one of two things:

```js
const activeExpandos = Array.from(primaryExpandos.values())
```

or keep using generators all the way down (with  `map`, `filter`, etc. from utils/generator.js):

```js
// Empty collapsed when beyond buffer
for (const expando of primaryExpandos.values()) {
/* ... */
primaryExpandos.values()
    ::filter(v => v.open)
    ::map(v => ({ media: v.media, data: v.media }))
    ::lazyUnload(isWithinBuffer);
```